device_doctor/bin/main.dart

```diff
@@ -5,6 +5,7 @@
 import 'package:args/args.dart';
 
 import 'package:device_doctor/src/device.dart';
+import 'package:device_doctor/src/ios_device.dart';
```

Comment:
Consider exporting what's needed from src, similar to how app_dart/lib/cocoon_service.dart works.

https://dart.dev/guides/language/effective-dart/usage#do-use-relative-paths-when-importing-libraries-within-your-own-packages-lib-directory

Reply:
Updated.